x/qgb/README.md

```diff
@@ -118,6 +118,42 @@ if k.CheckLatestAttestationNonce(ctx) && k.GetLatestAttestationNonce(ctx)+1 != n
 }
 ```
 
+## Verification command
+
+The QGB verification command is part of the `celestia-appd` binary. It allows the user to verify that a set of shares has been posted to a specific QGB contract.
+
+```shell
+$ celestia-appd verify --help          
```

Comment:
[non-blocking] This code snippet may become out of date with the true `--help` docs so instead of copy + pasting the entire help output, I think it may be preferable to include the text that isn't the code snippet and at the bottom say something like: 

See the following command for more details


```shell
celestia-appd verify --help
```

and then ensure that these docs make it into the appropriate `--help` docs

Reply:
What do you think is better for docs, providing the commands but without an output (that might be outdated after some time), or the commands + their output?

Reply:
> What do you think is better for docs, providing the commands but without an output, or the commands + their output (that might be outdated after some time)?

In this scenario: providing the commands but without an output. I think this because the goal of the README is to give developers a high level overview of the QGB module. Developers aren't necessarily users of the QGB module CLI. Users of the QGB CLI should be able to get information on CLI flags via `--help` and I expect them to try that before finding this README.

Also (repeated from above): the command output becomes a maintenance burden but I defer to you @SweeXordious

Reply:
I guess we can leave it as it is for now, with the help output. And, if it becomes a burden later, we can remove it.
Thanks guys for your help :+1: :+1: